session6/src/com/itschool/homework/HomerunChallenge7.java

```diff
@@ -0,0 +1,15 @@
+package com.itschool.homework;
+
+public class HomerunChallenge7 {
+    public static void main(String[] args) {
+        HomerunChallenge7 home = new HomerunChallenge7();
+        home.challengeRun7();
+    }
+    public void challengeRun7(){
+        int a = 2;
+        int b = 2;
+        int c = 2;
+        String triangleType1 = (a == b && b == c) ? "Equilateral" : (a == b || a == c || b == c) ? "Isoscel" : "Scalene";
```

Comment:
you did as per problem requirement, this is good. But remember that the ternary operator should be avoided in cases like this, since involves more than 3 expressions.